addons/web/static/src/views/list/list_view.js

```diff
@@ -21,13 +21,16 @@ export const listView = {
         const { ArchParser } = view;
         const { arch, relatedModels, resModel } = genericProps;
         const archInfo = new ArchParser().parse(arch, relatedModels, resModel);
+        const defaultGroupBy =
+            genericProps.searchMenuTypes.includes("groupBy") && archInfo.defaultGroupBy;
```

Comment:
Is it not better to put this logic directly in the list controller instead of passing it as a props ? All the needed information are available and it is simpler imo

Reply:
Indeed, I agree we can do it. 🙂

Reply:
However, in the props definition, we have a validation function as it is the case in the kanban view. I guess without passing it as a prop we will not have an error if the defaultGroupBy is not a string or falsy. 🤔 

What do you think? 🙂

Reply:
Oki thanks 🙂